cmd/internal/kube/template/akita-secret.tmpl

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: akita-secrets
+  namespace: {{.Namespace}}
```

Comment:
This should be optional; it need not be specified for the default namespace.

Reply:
I've made the namespace flag optional in [6596d5e](https://github.com/akitasoftware/akita-cli/pull/202/commits/6596d5e87dd4c624200a42f5bc4ef2a313b18fce)